### PR TITLE
1.0.0 Release

### PR DESCRIPTION
**Pull request by nblumhardt**
Very, very simple sink - probably nothing much to do before shipping the 1.0. Let's get it out there :-)

### CODE REVIEW

**Comment by nblumhardt** [comment]
..hold that, looks like compiling this in release mode ain't gonna fly :-)

**Comment by nblumhardt** [comment]
Pushed a fix; will double-check the dev package, this should be good to go.

**Comment by nblumhardt** [comment]
Dev package works. Could do additional work to try disabling the sink when the client app is not in debug mode, but probably not worth it since logging setup is environment/configuration specific anyway.

:shipit:

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+dotnet --info
+dotnet restore
+
+for path in src/**/*.csproj; do
+    dotnet build -f netstandard1.3 -c Release ${path}
+    dotnet build -f netcoreapp1.1 -c Release ${path}
```
Comment (khellang) [inline build.sh]: The target frameworks seems off here... There's no `netcoreapp1.1` *or* `netstandard1.3` projects under `src`.

**Review by khellang — DISMISSED** [review]

File: build.sh (hunk 1 above)
Comment (nblumhardt) [reply build.sh]: Copy-pasta; thanks 👍

**Comment by nblumhardt** [comment]
No Travis build set up on this one, so no guarantees it's working, but should have a better chance now :-)

**Pull request merged**